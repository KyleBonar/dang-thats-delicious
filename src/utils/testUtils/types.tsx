export interface MockLink {
  text: string;
  to: string;
  target: "_blank" | "_self";
}
