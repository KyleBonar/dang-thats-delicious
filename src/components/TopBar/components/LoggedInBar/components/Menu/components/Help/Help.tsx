import * as React from "react";

import Title from "../Title/Title";
import Item from "../Item/Item";

export interface HelpProps {}

const Help: React.FC<HelpProps> = props => {
  const _title = "Need Help?";
  return (
    <>
      <Title>{_title}</Title>
      <Item to="#">Quick Guide (Coming Soon)</Item>
      <Item to="#">Contact Us (Coming Soon)</Item>
    </>
  );
};

export default Help;
