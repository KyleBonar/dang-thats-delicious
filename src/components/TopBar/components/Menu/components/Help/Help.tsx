import * as React from "react";
import Title from "../Title/Title";
import Item from "../Item/Item";

export interface HelpProps {}

class Help extends React.PureComponent<HelpProps, any> {
  public render() {
    return (
      <div>
        <Title>Need Help?</Title>
        <Item to="#quickguide">Quick Guide</Item>
        <Item to="#contactus">Contact Us</Item>
      </div>
    );
  }
}

export default Help;
