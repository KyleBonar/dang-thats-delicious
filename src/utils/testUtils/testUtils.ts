import * as React from "react";
import casual from "casual";
import { ReactWrapper } from "enzyme";
import configureStore from "redux-mock-store";

import { MockCard } from "./types";
import { AppState } from "../../redux/configureStore";
import { ISauce, ISaucesState } from "../../redux/sauces/types";
import { FlashMessageProps } from "../../components/FlashMessage/FlashMessage";
import Flatn from "../Flatn/Flatn";
import { IUserState } from "../../redux/users/types";

export const ITERATION_SIZE = 32;
const REACT_REGEX = /react(\d+)?./i;

// seed it so we get consistent results
casual.seed(777);

const relatedSauce = () => ({
  name: casual.name,
  slug: casual.uuid
});

export const fakeSauce = (): ISauce => ({
  _id: casual.random_element([undefined, casual.integer(1, 500)]),
  _addedToStore: casual.random_element([undefined, casual.unix_time]), // Unix time (in seconds) added to redux store
  _full: casual.random_element([undefined, casual.boolean]), // Whether we have full sauce or partial
  _related: casual.random_element([
    undefined,
    new Array(ITERATION_SIZE).fill(null).map(relatedSauce)
  ]), // List of related sauces
  isAdminApproved: casual.random_element([undefined, casual.boolean]),
  name: casual.name,
  ingredients: casual.string,
  author: casual.name,
  created: casual.unix_time, // Unix time (in seconds)
  types: casual.array_of_words(),
  maker: casual.name,
  description: casual.string,
  photo: casual.random_element([undefined, casual.url]),
  shu: casual.random_element([
    undefined,
    casual.integer(1, 500),
    casual.integer(1, 500).toString()
  ]),
  reviews: casual.random_element([
    undefined,
    new Array(ITERATION_SIZE).fill(null).map(x => casual.word)
  ]),
  city: casual.random_element([undefined, casual.city]),
  state: casual.random_element([undefined, casual.state]),
  country: casual.random_element([undefined, casual.country]),
  slug: casual.random_element([undefined, casual.uuid])
});

export const fakeSaucesState = (): ISaucesState => {
  const sauces: ISauce[] = new Array(casual.integer(1, 15))
    .fill(null)
    .map(fakeSauce);
  const { allSlugs, bySlug } = Flatn.saucesArr({ sauces });

  return {
    allSlugs,
    bySlug,
    total: allSlugs.length,
    query: undefined,
    saucesWithNewestReviews: undefined,
    newest: undefined,
    featured: undefined,
    types: casual.array_of_words(),
    orders: ["Newest", "Name", "Times Reviewed", "Avg Rating"]
  };
};

const fakeUsersState = (): IUserState => ({
  self: casual.random_element([
    undefined,
    {
      token: casual.random_element([undefined, casual.uuid]),
      displayName: casual.random_element([undefined, casual.name]),
      avatarURL: casual.random_element([undefined, casual.url]),
      isAdmin: casual.random_element([undefined, casual.boolean])
    }
  ]),
  byDisplayName: undefined,
  allDisplayNames: undefined
});

export const fakeStore = () => {
  const storeConfig = configureStore([]);

  const store: AppState = {
    sauces: fakeSaucesState(),
    users: fakeUsersState(),
    reviews: {
      byReviewID: undefined,
      allReviewIDs: undefined
    }
  };

  return storeConfig(store);
};

export const fakeJSXElement = (): JSX.Element => {
  return casual.random_element([
    React.createElement("div", { key: casual.uuid }, casual.string),
    React.createElement("span", { key: casual.uuid }, casual.string)
  ]);
};

const fakeCard = (): MockCard => ({
  title: casual.string,
  description: casual.string,
  to: casual.url,
  showLink: casual.boolean,
  imageLink: casual.url,
  className: casual.string,
  anchorText: casual.string
});

const wait = (amount = 0) =>
  new Promise(resolve => setTimeout(resolve, amount));

function simulateInputChange(
  wrapper: ReactWrapper,
  name: string,
  value: string
) {
  wrapper
    // .find(`input[name='${name}']`)
    .simulate("change", { target: { name, value } });
}

const isClassComponent = (component): boolean => {
  return (
    typeof component === "function" &&
    component.prototype &&
    !!component.prototype.isReactComponent
  );
};

// Ensure compatability with transformed code
const isFunctionComponent = (component): boolean => {
  return (
    typeof component === "function" &&
    String(component).includes("return") &&
    !!String(component).match(REACT_REGEX) &&
    String(component).includes(".createElement")
  );
};

const isElement = (typeElement): boolean => {
  return React.isValidElement(typeElement);
};

const isComponent = (component): boolean => {
  return isClassComponent(component) || isFunctionComponent(component);
};

const isDOMTypeElement = (typeElement): boolean => {
  return isElement(typeElement) && typeof typeElement.type === "string";
};

const generateValidPassword = (MIN_PASSWORD_LENGTH: number = 8) => {
  // generate length between minimum length and 3x the minimum length
  const _randomLength = casual.integer(
    MIN_PASSWORD_LENGTH,
    3 * MIN_PASSWORD_LENGTH
  );

  // turn generated length into random letters
  return new Array(_randomLength).fill(null).map(casual._letter).join("");
};

const generateInValidPassword = (MIN_PASSWORD_LENGTH: number = 8) => {
  // generate length between minimum length and 3x the minimum length
  const _randomLength = casual.integer(0, MIN_PASSWORD_LENGTH);

  // turn generated length into random letters
  return new Array(_randomLength).fill(null).map(casual._letter).join("");
};

export {
  casual,
  simulateInputChange,
  isDOMTypeElement,
  isComponent,
  wait,
  generateValidPassword
};
