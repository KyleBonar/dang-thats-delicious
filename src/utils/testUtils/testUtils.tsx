import * as React from "react";
import * as enzyme from "enzyme";
import casual from "casual";
import { ReactWrapper } from "enzyme";
import configureStore from "redux-mock-store";
import thunk from "redux-thunk";

import { AppState } from "../../redux/configureStore";
import { ISauce, ISaucesState } from "../../redux/sauces/types";
import Flatn from "../Flatn/Flatn";
import { IUserState } from "../../redux/users/types";
import { IErrReturn } from "../Err/Err";
import { act } from "react-dom/test-utils";
import { Provider } from "react-redux";

export const ITERATION_SIZE = 32;
const REACT_REGEX = /react(\d+)?./i;

// seed it so we get consistent results
casual.seed(777);

const relatedSauce = () => ({
  name: casual.name,
  slug: casual.uuid
});

export const fakeSauce = (): ISauce => ({
  _id: casual.random_element([undefined, casual.integer(1, 500)]),
  _addedToStore: casual.random_element([undefined, casual.unix_time]), // Unix time (in seconds) added to redux store
  _full: casual.random_element([undefined, casual.boolean]), // Whether we have full sauce or partial
  _related: casual.random_element([
    undefined,
    new Array(ITERATION_SIZE).fill(null).map(relatedSauce)
  ]), // List of related sauces
  isAdminApproved: casual.random_element([undefined, casual.boolean]),
  name: casual.name,
  ingredients: casual.random_element([undefined, casual.string]),
  author: casual.name,
  created: casual.unix_time, // Unix time (in seconds)
  types: casual.random_element([undefined, casual.array_of_words()]),
  maker: casual.name,
  description: casual.string,
  photo: casual.random_element([undefined, casual.url]),
  shu: casual.random_element([
    undefined,
    casual.integer(1, 500),
    casual.integer(1, 500).toString()
  ]),
  reviews: casual.random_element([
    undefined,
    new Array(ITERATION_SIZE).fill(null).map(x => casual.word)
  ]),
  city: casual.random_element([undefined, casual.city]),
  state: casual.random_element([undefined, casual.state]),
  country: casual.random_element([undefined, casual.country]),
  slug: casual.random_element([undefined, casual.uuid])
});

export const generateFakeSauces = (): ISauce[] =>
  new Array(casual.integer(1, 15)).fill(null).map(fakeSauce);

export const fakeSaucesState = (): ISaucesState => {
  const sauces: ISauce[] = generateFakeSauces();
  const { allSlugs, bySlug } = Flatn.saucesArr({ sauces });

  const featured: undefined | string[] = casual.random_element([
    undefined,
    allSlugs.length > 0
      ? allSlugs.filter(slug => {
          if (casual.random < 0.4) return slug;
          return;
        })
      : undefined
  ]);

  return {
    allSlugs,
    bySlug,
    total: allSlugs.length,
    query: undefined,
    saucesWithNewestReviews: undefined,
    newest: undefined,
    featured,
    types: casual.array_of_words(),
    orders: ["Newest", "Name", "Times Reviewed", "Avg Rating"]
  };
};

const fakeUsersState = (): IUserState => ({
  self: casual.random_element([
    undefined,
    {
      token: casual.random_element([undefined, casual.uuid]),
      displayName: casual.random_element([undefined, casual.name]),
      avatarURL: casual.random_element([undefined, casual.url]),
      isAdmin: casual.random_element([undefined, casual.boolean])
    }
  ]),
  byDisplayName: undefined,
  allDisplayNames: undefined
});

export const fakeStore = () => {
  const middlewares = [thunk];
  const storeConfig = configureStore(middlewares);

  const store: AppState = {
    sauces: fakeSaucesState(),
    users: fakeUsersState(),
    reviews: {
      byReviewID: undefined,
      allReviewIDs: undefined
    }
  };

  return storeConfig(store);
};

export const fakeJSXElement = (): JSX.Element => {
  return casual.random_element([
    React.createElement("div", { key: casual.uuid }, casual.string),
    React.createElement("span", { key: casual.uuid }, casual.string)
  ]);
};

const wait = (amount = 0) =>
  new Promise(resolve => setTimeout(resolve, amount));

function simulateInputChange(
  wrapper: ReactWrapper,
  name: string,
  value: string
) {
  wrapper
    // .find(`input[name='${name}']`)
    .simulate("change", { target: { name, value } });
}

const isClassComponent = (component): boolean => {
  return (
    typeof component === "function" &&
    component.prototype &&
    !!component.prototype.isReactComponent
  );
};

// Ensure compatability with transformed code
const isFunctionComponent = (component): boolean => {
  return (
    typeof component === "function" &&
    String(component).includes("return") &&
    !!String(component).match(REACT_REGEX) &&
    String(component).includes(".createElement")
  );
};

const isElement = (typeElement): boolean => {
  return React.isValidElement(typeElement);
};

const isComponent = (component): boolean => {
  return isClassComponent(component) || isFunctionComponent(component);
};

const isDOMTypeElement = (typeElement): boolean => {
  return isElement(typeElement) && typeof typeElement.type === "string";
};

const generateValidPassword = (MIN_PASSWORD_LENGTH: number = 8): string => {
  // generate length between minimum length and 3x the minimum length
  const _randomLength = casual.integer(
    MIN_PASSWORD_LENGTH,
    3 * MIN_PASSWORD_LENGTH
  );

  // turn generated length into random letters
  return new Array(_randomLength).fill(null).map(casual._letter).join("");
};

const generateInValidPassword = (MIN_PASSWORD_LENGTH: number = 8): string => {
  // generate length between minimum length and 3x the minimum length
  const _randomLength = casual.integer(1, MIN_PASSWORD_LENGTH - 1);

  // turn generated length into random letters
  return new Array(_randomLength).fill(null).map(casual._letter).join("");
};

// Function designed to resemble how a network error would look like.
// This will help the catch(err) only have to handle a single format
const generateErr = (): IErrReturn => {
  return {
    response: {
      data: {
        status: 400,
        msg: casual.string,
        isGood: false
      }
    }
  };
};

// lets us mount up a simple custom hook
const mountReactHook = hook => {
  const Component = ({ children }) => children(hook());
  const componentHook = {};
  let componentMount;

  act(() => {
    componentMount = enzyme.shallow(
      <Component>
        {hookValues => {
          Object.assign(componentHook, hookValues);
          return null;
        }}
      </Component>
    );
  });
  return { componentMount, componentHook };
};

export interface ImountReactHookWithReduxStore {
  componentMount: any;
  componentHook: any;
}
const mountReactHookWithReduxStore = (
  hook,
  store
): ImountReactHookWithReduxStore => {
  const Component = ({ children }) => children(hook());
  const componentHook = {};
  let componentMount;

  act(() => {
    componentMount = enzyme.mount(
      <Provider store={store}>
        <Component>
          {hookValues => {
            Object.assign(componentHook, hookValues);
            return null;
          }}
        </Component>
      </Provider>
    );
  });
  return { componentMount, componentHook };
};

export {
  casual,
  simulateInputChange,
  isDOMTypeElement,
  isComponent,
  wait,
  generateValidPassword,
  generateInValidPassword,
  generateErr,
  mountReactHook,
  mountReactHookWithReduxStore
};
