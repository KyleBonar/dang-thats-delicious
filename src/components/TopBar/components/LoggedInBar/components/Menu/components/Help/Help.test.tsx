import * as React from "react";
import * as enzyme from "enzyme";

import Help from "./Help";
import { ITERATION_SIZE } from "../../../../../../../../utils/testUtils/testUtils";

describe("<Help />", () => {
  const _title = "Need Help?";

  it("renders", () => {
    new Array(ITERATION_SIZE).fill(null).map(() => {
      const wrapper = enzyme.shallow(<Help />);

      expect(wrapper).toBeTruthy();
    });
  });

  it("matches snapshot", () => {
    new Array(ITERATION_SIZE).fill(null).map(() => {
      const wrapper = enzyme.shallow(<Help />);

      expect(wrapper).toMatchSnapshot();
    });
  });

  it("renders a Title component", () => {
    new Array(ITERATION_SIZE).fill(null).map(() => {
      const wrapper = enzyme.shallow(<Help />);

      expect(wrapper.find("Title").exists()).toBeTruthy();
    });
  });

  it("passes expected value to Title", () => {
    new Array(ITERATION_SIZE).fill(null).map(() => {
      const wrapper = enzyme.shallow(<Help />);

      expect(wrapper.find("Title").prop("children")).toEqual(_title);
    });
  });

  it("renders two Item components", () => {
    new Array(ITERATION_SIZE).fill(null).map(() => {
      const wrapper = enzyme.shallow(<Help />);

      expect(wrapper.find("Item").length).toEqual(2);
    });
  });
});
