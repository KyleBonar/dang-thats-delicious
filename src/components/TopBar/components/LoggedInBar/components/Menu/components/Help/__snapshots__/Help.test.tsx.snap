// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Help /> matches snapshot 1`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 2`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 3`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 4`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 5`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 6`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 7`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 8`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 9`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 10`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 11`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 12`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 13`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 14`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 15`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 16`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 17`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 18`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 19`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 20`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 21`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 22`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 23`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 24`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 25`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 26`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 27`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 28`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 29`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 30`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 31`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;

exports[`<Help /> matches snapshot 32`] = `
<Fragment>
  <Title>
    Need Help?
  </Title>
  <Item
    to="#"
  >
    Quick Guide (Coming Soon)
  </Item>
  <Item
    to="#"
  >
    Contact Us (Coming Soon)
  </Item>
</Fragment>
`;
